protocols/cosi/packets.go

```diff
@@ -15,6 +15,17 @@ import (
 type CosiAnnouncement struct {
 	// From = TreeNodeId in the Tree
 	From uuid.UUID
+	// Tells to the node whether they should make the verification or not
+	// XXX XXX UGLY HACK XXX XXX
+	// Is to be removed during the next versions of SDA. Need that for the
+	// moment because no way to have external configuration for the intermediate
+	// and leaf nodes.
+	// see https://github.com/dedis/cothority/issues/260
+	// 0 - no check at all
+	// 1 - check only at root
+	// 2 - check at each level of the tree
+	VerifyResponse int
+
```

Comment:
Instead of giving it as a parameter in the message, you can also use a setter and call that setter in https://github.com/dedis/cothority/blob/master/lib/sda/simulation.go#L43 - which will be somewhere in https://github.com/dedis/cothority/blob/check/protocols/cosi/simulation.go#L83 - if you do that, be sure to call

```
cs.SimulationBFTree.Node( config )
```

before, so that the tree and the entityList are set up.